lib/Runtime/Library/JavascriptNumber.cpp

```diff
@@ -230,6 +230,14 @@ namespace Js
 
     static const double d1_0 = 1.0;
 
+#ifdef __clang__
+    // todo : make this fast!
+    // Clang doesn't support so many things under the fast version below
```

Comment:
It is not about msse3 etc. Clang inline assembly does not support non-asm in naked function.